dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/enums/data/DatasetQualityStatus.kt

```diff
@@ -0,0 +1,11 @@
+package org.dataland.datalandbackend.model.enums.data
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+/**
+ * An enum with the different stati of review a dataset has
+ */
+@Schema(
+    enumAsRef = true,
+)
+enum class DatasetQualityStatus { Pending, Accepted }
```

Comment:
I think this should be a `DatasetQAStatus` or `DatasetQualityAssuranceStatus`.

Reply:
I see. How about only `QAStatus`?

Reply:
yes - sounds good.

Reply:
ok